test/tests/clone.js

```diff
@@ -286,6 +289,66 @@ describe("Clone", function() {
     });
   });
 
+  if (process.platform === "win32") {
+    it("can clone with ssh using old agent with sha1 signing support only",
+      async function () {
+      var pageant = local("../../vendor/pageant.exe");
+      var old_pageant = local("../../vendor/pageant_sha1.exe");
+      var privateKey = local("../../vendor/private.ppk");
+      var test = this;
+      var url = "git@github.com:nodegit/test.git";
+      var opts = {
+        fetchOpts: {
+          callbacks: {
+            certificateCheck: () => 0,
+            credentials: function(url, userName) {
+              return NodeGit.Credential.sshKeyFromAgent(userName);
+            }
+          }
+        }
+      };
+
+      try {
+        await exec("taskkill /im pageant.exe /f /t");
+      } catch (e) {
+        try {
+          await exec("taskkill /im pageant_sha1.exe /f /t");
+        } catch(e) {}
+        try {
+          await exec(pageant, [privateKey]);
```

Comment:
It looks like elsewhere we ```await exec(`powershell -command "Start-Process ${old_pageant} ${privateKey}`);```, should we be doing that here as well? Or should we be doing this in the other locations?

Reply:
my bad, I will fix it